hlint.cabal

```diff
@@ -67,14 +67,15 @@ library
         extra >= 1.6.6,
         refact >= 0.3,
         aeson >= 1.1.2.0,
-        filepattern >= 0.1.1
+        filepattern >= 0.1.1,
+        ghc-lib-parser-ex == 8.8.2
     if !flag(ghc-lib) && impl(ghc >= 8.8.0) && impl(ghc < 8.9.0)
         build-depends:
           ghc == 8.8.*,
           ghc-boot-th
     else
         build-depends:
-          ghc-lib-parser == 8.8.*
+          ghc-lib-parser == 8.8.2
```

Comment:
Ditto

Reply:
Yes. See above.

Reply:
If we follow my suggestion above this can stay at `8.8.*`

Reply:
Yes, see below.